components/dashboard/src/components/error-boundaries/ReloadPageErrorBoundary.tsx

```diff
@@ -56,7 +56,9 @@ export const ReloadPageErrorFallback: FC<Pick<FallbackProps, "error">> = ({ erro
                         <strong>Code:</strong> {caughtError.code}
                     </span>
                 )}
-                {caughtError.message && <pre>{caughtError.message}</pre>}
+                {caughtError.message && (
```

Comment:
Fixing some styles on the error message so it wraps now instead of flowing off the page if it's too long.